bundles/core/org.eclipse.smarthome.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/XmlThingTypeProvider.java

```diff
@@ -73,9 +83,11 @@ public XmlThingTypeProvider() {
      * specified module.
      * <p>
      * This method returns silently, if any of the parameters is {@code null}.
-     *
-     * @param bundle the module to which the Thing type to be added
-     * @param thingType the Thing type to be added
+     * 
+     * @param bundle
+     *            the module to which the Thing type to be added
+     * @param thingType
```

Comment:
Did the original ESH formatter change this? Then the question is again, why it was not formatted correctly before.